src/doc/src/reference/build-script-examples.md

```diff
@@ -440,7 +440,7 @@ script looks something [like
 this](https://github.com/sfackler/rust-openssl/blob/dc72a8e2c429e46c275e528b61a733a66e7877fc/openssl-sys/build/main.rs#L216):
 
 ```rust,ignore
-println!("cargo::version_number={:x}", openssl_version);
+println!("cargo:version_number={openssl_version:x}");
```

Comment:
Thanks for fixing the doc!

This is an example that uses the new double-colons syntax (`cargo::`), it was an overlook and should be

```suggestion
println!("cargo::metadata=version_number={openssl_version:x}");
```

See <https://doc.rust-lang.org/nightly/cargo/reference/build-scripts.html#the-links-manifest-key>.